dask/dataframe/core.py

```diff
@@ -1605,6 +1605,36 @@ def last(self, offset):
                           date, None, True, False, 'ix')
         return new_dd_object(merge(self.dask, dsk), name, self, divs)
 
+    def nunique_approx(self, b=16):
+        """Approximate number of unique rows.
+
+        This method uses the HyperLogLog algorithm for cardinality
+        estimation to compute the approximate number of rows.
+
+        Parameters
+        ----------
+        b : integer, default 16
+            The number of bits of the hash to use for approximating
+            the number of unique elements. Large values of b require more
+            time and memory to compute. b should be between 8 and 16 (inclusive)
+            for the current implementation. The relative error in the
+            number of elements is approximated by 1.04 / sqrt(2**b)
+
+        Returns
+        -------
+        a float representing the approximate number of elements
+        """
+
+        from . import _hyperloglog # here to avoid circular import issues
+
+        return aca(
+            [self],
+            chunk=_hyperloglog.compute_hll_array,
+            aggregate=_hyperloglog.estimate_count,
+            split_every=False,
+            b=b,
+            meta=float)
```

Comment:
You can in principle. It's just the maximum value at each index.

The reason I hard-coded `split_every=False`, IIRC, was that originally `aca` didn't have it, so I wrote it without, then when I rebased a newer master I just did the easiest thing to get it running. To keep calling aggregate, we'd need to keep re-maxing the state array, not the count...this shouldn't be too hard, though.